app/src/main/kotlin/Navigation.kt

```diff
@@ -64,8 +65,14 @@ val screens = mutableMapOf(
             "Dynamic Price Banner + Video",
             "Dynamic Price Inline Video",
             "Dynamic Price Interstitial",
+        )),
+        NavigationAdapter(destination = "AdMob Dynamic Price", header = "AdMob", items = arrayOf(
```

Comment:
We should not references to `AdMob Dynamic Price` in code. Please rename it to `Dynamic Adapters for AdMob`

Reply:
addressed in https://github.com/timehop/nimbus-android-sample/pull/52